fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -161,6 +161,7 @@ public class OperationType {
     public static final short OP_SAVE_TRANSACTION_ID = 104;
     public static final short OP_SAVE_AUTO_INCREMENT_ID = 105;
     public static final short OP_DELETE_AUTO_INCREMENT_ID = 106;
+    public static final short OP_AUTO_INCREMENT_FORCE_DELETE = 107;
 
```

Comment:
newly added OP id should start with 10000, refer to OP_MODIFY_TABLE_CONSTRAINT_PROPERTY.